install_circular_subspec_dependency/after/Pods/Target Support Files/Pod1/Pod1.xcconfig

```diff
@@ -1,10 +1,11 @@
 CONFIGURATION_BUILD_DIR = ${PODS_CONFIGURATION_BUILD_DIR}/Pod1
 GCC_PREPROCESSOR_DEFINITIONS = $(inherited) COCOAPODS=1
-HEADER_SEARCH_PATHS = "${PODS_ROOT}/Headers/Private" "${PODS_ROOT}/Headers/Private/Pod1" "${PODS_ROOT}/Headers/Public" "${PODS_ROOT}/Headers/Public/Pod1" "${PODS_ROOT}/Headers/Public/Pod2"
+HEADER_SEARCH_PATHS = $(inherited) "${PODS_ROOT}/Headers/Private" "${PODS_ROOT}/Headers/Private/Pod1" "${PODS_ROOT}/Headers/Public" "${PODS_ROOT}/Headers/Public/Pod1" "${PODS_ROOT}/Headers/Public/Pod2"
 LIBRARY_SEARCH_PATHS = $(inherited) "${PODS_CONFIGURATION_BUILD_DIR}/Pod2"
 PODS_BUILD_DIR = ${BUILD_DIR}
 PODS_CONFIGURATION_BUILD_DIR = ${PODS_BUILD_DIR}/$(CONFIGURATION)$(EFFECTIVE_PLATFORM_NAME)
 PODS_ROOT = ${SRCROOT}
 PODS_TARGET_SRCROOT = ${PODS_ROOT}/../DeveloperPods/Pod1
 PRODUCT_BUNDLE_IDENTIFIER = org.cocoapods.${PRODUCT_NAME:rfc1034identifier}
 SKIP_INSTALL = YES
+SWIFT_INCLUDE_PATHS = $(inherited) "${PODS_CONFIGURATION_BUILD_DIR}/Pod2"
```

Comment:
why would this show up in the diff if the Pod2 does not have Swift in it? Does it?

Reply:
What I am asking is if we want all of these to be added always or just when the target pod `uses_swift?`?

Reply:
it isnt strictly necessary since `Pod1` does not have any swift, but it doesnt harm anything, same as setting `GCC_PREPROCESSOR_DEFINITIONS` for an all-ObjC pod